.github/workflows/test.yml

```diff
@@ -20,6 +20,11 @@ jobs:
           - head
           - truffleruby-head
         os: [ubuntu-latest]
+        with_latest_reline: [true, false]
+        exclude:
+          - ruby: '2.5'
```

Comment:
I plan to drop the support of Ruby 2.5 because it won't be supported by the next `reline` release.

Reply:
We can probably drop support for 2.6 and earlier completely since they aren't supported Rubies anymore.

Reply:
I don't mind keeping 2.6 available until the day we need to make workaround for it

Reply:
We can probably completely drop support for 2.5, it EOL'd in March 2021, almost 1.5 years ago.

Reply:
It requires other changes like gemspec so I’ll drop it in another PR 👍